packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -316,13 +316,23 @@ class _GameWidgetState<T extends Game> extends State<GameWidget<T>> {
             color: game.backgroundColor(),
             child: LayoutBuilder(
               builder: (_, BoxConstraints constraints) {
-                game.onGameResize(constraints.biggest.toVector2());
+                final size = constraints.biggest.toVector2();
+                if (size.isZero()) {
+                  return widget.loadingBuilder?.call(context) ?? Container();
+                }
+                game.onGameResize(size);
                 return FutureBuilder(
                   future: loaderFuture,
                   builder: (_, snapshot) {
                     if (snapshot.hasError) {
                       final errorBuilder = widget.errorBuilder;
                       if (errorBuilder == null) {
+                        // @Since('2.16')
+                        // throw Error.throwWithStackTrace(
+                        //   snapshot.error!,
+                        //   snapshot.stackTrace,
+                        // )
+                        print(snapshot.stackTrace);
```

Comment:
I guess this was accidentally left?

Reply:
No, this was deliberate. Right now if an error occurs during loading, the stacktrace is lost. With `print`, it would at least be in the console. The best solution is to re-throw with stacktrace, but that function is only available in Dart 2.16. So I left the comment here to remember to improve this part once we upgrade to a newer Dart version.

Reply:
I would at least use `log` from `dart:developer` here instead

Reply:
We should also add a `NOTE:` in the code then to keep track of it OR make a follow up issue for it :D

Reply:
Or update to Dart 2.16...

Reply:
Current Flutter version uses Dart 2.15; but Flutter 2.10 (currently in beta) will have Dart 2.16. So, hopefully, we'll be able to upgrade soon. In the meanwhile, i'll post an issue.